src/o365/teams/commands/report/teams-report-useractivityuserdetail.ts

```diff
@@ -65,6 +102,15 @@ class TeamsReportUserActivityUserDetailCommand extends GraphCommand {
       {
         option: '-d, --date [date]',
         description: 'The date for which you would like to view the users who performed any activity. Supported date format is YYYY-MM-DD. Specify the date or period, but not both.'
+      },
+      {
+        option: '-f, --outputFile [outputFile]',
+        description: 'Path to the file where the upgrade report should be stored in.'
+      },
+      {
+        option: '-o, --output [output]',
```

Comment:
This option already exists in parent class. We should not override it here.

Reply:
I have fixed this in the latest commit.